frontend/src/style/Tools.css

```diff
@@ -0,0 +1,32 @@
+.maintitle {
```

Comment:
I think that the css might be misaligning things here. Rerun the code and make sure that the spacing for the title looks ok. It does not appear to be centered.

Reply:
I think I figured out the issue. I will leave some comments here down below.